Sources/Controllers/MessagesViewController.swift

```diff
@@ -235,14 +250,14 @@ extension MessagesViewController: UICollectionViewDataSource {
 
 // MARK: - Keyboard Handling
 
-extension MessagesViewController {
+fileprivate extension MessagesViewController {
 
```

Comment:
What are these notifications used for

Reply:
Keyboard handling 🤔

Reply:
I know what they are for but it should be documented no?

Reply:
Ah, sorry I misunderstood. I couldn't add anymore information than the method name itself lol. The function is self documenting 😂

Reply:
But then jazzy won't report 100% doc coverage :(

Reply:
I'm hoping that's not the case, because I don't want Jazzy documenting the internal API.